spring-cloud-skipper-server/src/main/java/org/springframework/cloud/skipper/server/deployer/AppDeploymentRequestFactory.java

```diff
@@ -60,8 +79,7 @@ public AppDeploymentRequest createAppDeploymentRequest(SpringBootAppKind springB
 			deploymentProperties.put(AppDeployer.COUNT_PROPERTY_KEY, String.valueOf(metadata.get("count")));
 		}
 
-		// TODO fix setting of group property with new version, will mess up consumer groups.
-		deploymentProperties.put(AppDeployer.GROUP_PROPERTY_KEY, releaseName + "-v" + version);
+		deploymentProperties.put(AppDeployer.GROUP_PROPERTY_KEY, releaseName);
```

Comment:
What happens in the case when the deployment properties has already set the `group` in it?

Reply:
At least in SCDF, we always set the `group` to the stream name. Do we want to override that here?

Reply:
when deploying via SCDF I believe we always want to use the group name to be the `streamName` as SCDF sets it every time the stream is deployed. Not sure if we want to override it with the `releaseName` when deploying via Skipper because, it potentially changes the group for each `release` the SCDF stream gets deployed/updated/rolledback.

Maybe we could only set the `group` if the deployment properties doesn’t have it set already?

Reply:
Yes, I think that is correct.  If the deployment group property is already set, as it will be when the package it uploaded to skipper from SCDF, we just use that.  Forgot to consider this case.  I'll make the change and merge.